docs/source/release_notes/release-notes.rst

```diff
@@ -35,41 +35,64 @@ Changelog
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1341>`__)
 
     *    - |improved|
-         - Oracle query performance improvement; table name cache now global to KVS level.
+         - Increased connection pool idle timeout to 10 minutes, and reduced eviction check frequency to 20-30 seconds at 1/10 of connections.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1336>`__)
+
+    *    - |new|
+         - There is a new configuration called ``maxConnectionBurstSize``, which configures how large the pool is able to grow when receiving a large burst of requests.
+           Previously this was hard-coded to 5x the ``poolSize`` (which is now the default for the parameter).
+
+           See :ref:`Cassandra KVS Config <cassandra-kvs-config>` for details on configuring AtlasDB with Cassandra.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1336>`__)
+
+    *    - |improved|
+         - Improved the performance of Oracle queries by making the table name cache global to the KVS level.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1325>`__)
 
     *    - |fixed|
-         - Oracle value style caching limited in scope to per-KVS, previously per-JVM,
-           which could have in extremely rare cases caused issues for users in odd configurations.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1235>`__)
+         - Oracle value style caching limited in scope to per-KVS, previously per-JVM, which could have in extremely rare cases caused issues for users in non-standard configurations.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1325>`__)
 
     *    - |new|
-         - We now publish a runnable distribution of AtlasCli that is available for download directly from Bintray
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1345>`__)
+         - We now publish a runnable distribution of AtlasCli that is available for download directly from `Bintray <https://bintray.com/palantir/releases/atlasdb#files/com/palantir/atlasdb/atlasdb-cli-distribution>`__.
+           (`Pull Request 1 <https://github.com/palantir/atlasdb/pull/1318>`__) and
+           (`Pull Request 2 <https://github.com/palantir/atlasdb/pull/1345>`__)
 
     *    - |improved|
-         - Enable garbage collection logging for CircleCI builds.
+         - Enabled garbage collection logging for CircleCI builds.
+           This may be useful for investigating pre-merge build failures.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1398>`__)
 
-    *    - |new|
-         - AtlasDB now supports stream store compression.
-           Streams can be compressed client-side by adding the ``compressStreamInClient`` option to the stream 
-           definition. Reads from the stream store will transparently decompress the data.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1357>`__)
+    *    - |changed|
```

Comment:
The newer version includes the fix for a [bug](https://github.com/palantir/gradle-java-distribution/issues/144) filed by our very own @SerialVelocity, so I'm guessing that's the motivation. 

It helps our ETE tests to be more stable. That isn't noteworthy in itself, but products depending on both AtlasDB and gradle-java-distribution could be affected.